src/test/scala/viper/gobra/GobraPackageTests.scala

```diff
@@ -67,6 +68,7 @@ class GobraPackageTests extends GobraTests {
           reporter = NoopReporter,
           inputFiles = input.files.toVector,
           includeDirs = Vector(currentDir),
+          checkConsistency = true,
```

Comment:
Just as an observation, this may cause some tests to fail in the CI due to problems not related to the changes in this PR.